JavaScript/JSX Comments.tmPreferences

```diff
@@ -5,7 +5,7 @@
     <key>name</key>
     <string>JSX Comments</string>
     <key>scope</key>
-    <string>source.jsx meta.jsx - source.jsx.embedded.jsx, source.ts meta.jsx - source.ts.embedded.jsx</string>
+    <string>source.jsx meta.jsx - source.jsx.embedded.jsx, source.ts meta.jsx - source.ts.embedded.jsx, source.tsx meta.jsx - source.tsx.embedded.jsx</string>
```

Comment:
With the restructuring, we not longer have `jsx` in `ts`, only `tsx`, so I think the suggestion below is more correct, right?

```suggestion
    <string>source.jsx meta.jsx - source.jsx.embedded.jsx, source.tsz meta.jsx - source.tsx.embedded.jsx</string>
```

Reply:
@Thom1729?